py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -51,7 +51,7 @@ def sum(
     ):
         input_val = cast_trt_tensor(ctx, input_val, trt.float32, name)
 
-    if dim is None:
+    if dim is None or (isinstance(dim, (tuple, list)) and len(dim) == 0):
```

Comment:
Should we consider `dim` is [] or ()? In this case, the dim passed into axes will be 0.

Reply:
I thought that `dim` being empty implies the same thing as `dim` being None, which is to reduce over all axes

Reply:
This is the behavior in `torch`, at least with `torch.sum`